modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/model/BookmarksSortTypeForProfile.kt

```diff
@@ -26,7 +26,7 @@ enum class BookmarksSortTypeForProfile(
     PODCAST_AND_EPISODE(
         labelId = R.string.podcast_and_episode,
         key = SORT_TYPE_PODCAST_AND_EPISODE,
-        serverId = 2,
+        serverId = 4, // Matches the serverId for the same sort type in iOS app
```

Comment:
AFAIK Web uses `2` as a value.

Reply:
I checked iOS implementation [here](https://github.com/Automattic/pocket-casts-ios/blob/6a41c5a94dde471f4193593081a5a656ccc6254c/Modules/DataModel/Sources/PocketCastsDataModel/Public/Enums.swift#L79)  which sets podcast and episode sort type value as 4. I'll cross-check again tomorrow and update.